.buckconfig.local

```diff
@@ -0,0 +1,15 @@
+[buildfile]
```

Comment:
We cannot check in .buckconfig since it will affect internal repo (when user is in under xplat/caffe2 folder). But .buckconfig.local is fine since it will not take effect without .buckconfig file. That's why I check in this file and create an empty .buckconfig during CI run. Alternatively I can add a flag file but not that convenient.

Reply:
Oh, bummer.  What if we call this .buckconfig.oss, and have the setup script copy it to .buckconfig?  That way it's still possible to have a .buckconfig.local?